histogram_test.go

```diff
@@ -86,7 +86,7 @@ func TestBucketPairsSortsDurationBuckets(t *testing.T) {
 	assert.Equal(t, time.Duration(math.MaxInt64), pairs[3].UpperBoundDuration())
 }
 
-func TestBucketPairsDurationBucketsInsertsMissingZero(t *testing.T) {
+func TestBucketPairsDurationBucketsDoesntInsertMissingZero(t *testing.T) {
```

Comment:
Probably can delete this test, it was mainly for branch coverage and verification of an added feature.  We don't really need to test the non-existence of features.